src/test/py/bazel/bzlmod/bazel_lockfile_test.py

```diff
@@ -1741,19 +1742,19 @@ def testExtensionRepoMappingChange_loadsAndRepoRelativeLabels(self):
 
     # Shutdown bazel to make sure we rely on the lockfile and not skyframe
     self.RunBazel(['shutdown'])
-    # Now, for something spicy: upgrade foo to 2.0 and change nothing else.
+    # Now, for something spicy: change repo_name to point to bar.
     # The extension should rerun despite the lockfile being present, and no
     # usages or .bzl files having changed.
     self.ScratchFile(
         'MODULE.bazel',
         [
-            'bazel_dep(name="foo",version="2.0")',
+            'bazel_dep(name="bar",version="1.0",repo_name="repo_name")',
             'ext = use_extension(":ext.bzl", "ext")',
             'use_repo(ext, "repo")',
         ],
     )
     _, _, stderr = self.RunBazel(['build', ':lol'])
-    self.assertIn('STR=@@foo~2.0//:BUILD', '\n'.join(stderr))
+    self.assertIn('STR=@@bar~//:BUILD', '\n'.join(stderr))
 
   def testExtensionRepoMappingChange_sourceRepoNoLongerExistent(self):
```

Comment:
@Wyverald Do you happen to have an idea how to port this test over to the new naming scheme?

Reply:
worst case scenario, we could manually alter the recorded repo mapping entries to contain a nonexistent source repo. that would probably simplify the test setup a lot too.